pytest_html/resources/main.js

```diff
@@ -160,33 +156,31 @@ function toggle_sort_states(elem) {
     }
 }
 
-function split_extra_onto_two_rows() {
-    find_all('tr.results-table-row').forEach(function(elem) {
-        var new_row = document.createElement("tr")
-        new_row.className = "extra";
-        elem.parentNode.insertBefore(new_row, elem.nextSibling);
-        find_all(".extra", elem).forEach(function (td_elem) {
-            if (find("*:not(.empty)", td_elem)) {
-                new_row.appendChild(td_elem);
-                td_elem.colSpan=5;
-            } else {
-                td_elem.parentNode.removeChild(td_elem);
-            }
-        });
-    });
+function is_all_rows_hidden() {
+    var rows = find_all('.results-table-row');
+    if(rows.length == 0)
```

Comment:
Could you use a [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) here, to reduce the array to only rows that are not hidden. Then you can just return the result of checking the length against zero.